Makefile

```diff
@@ -12,7 +12,6 @@ endif
 test: clean
 	./manage.py test --settings=$(test_settings) --with-coverage --with-ignore-docstrings \
 		--exclude-dir=notesserver/settings --cover-inclusive --cover-branches \
-		--ignore-files=search_indexes.py --ignore-files=highlight.py\
```

Comment:
django-nose is having issue with `--ignore-files`. https://github.com/django-nose/django-nose/issues/240